ReadMe.md

```diff
@@ -6,7 +6,7 @@
 [![Telegram](https://img.shields.io/badge/telegram-join%20chat-3796cd.svg)](https://t.me/clickhousebackup)
 [![Docker Image](https://img.shields.io/docker/pulls/alexakulov/clickhouse-backup.svg)](https://hub.docker.com/r/alexakulov/clickhouse-backup)
 
-Tool for easy ClickHouse backup and restore with cloud storages support
+Tool for easy ClickHouse backup and restore with cloud storage support
```

Comment:
it was means multiple cloud storage types

Reply:
I think storage is usually uncountable, we can say storage and mean multiple storage types. But it's really up to you, if you like storages, I can revert this change

Reply:
Ok. We need something like `with support many cloud storage types`